src/Alpha_complex/doc/Intro_alpha_complex.h

```diff
@@ -146,6 +158,28 @@ namespace alpha_complex {
  * `SimplicialComplexForAlpha::prune_above_filtration()`).
  * In the following example, the value is given by the user as argument of the program.
  *
+ * \section weightedversion Weighted specific version
+ * <b>Requires:</b> \ref eigen &ge; 3.1.0 and \ref cgal &ge; 5.1.0.
+ * For performances reasons, it is advised to use \ref eigen &ge; 3.5.8 and \ref cgal &ge; 5.2.0
```

Comment:
Do you remember where the 3.5.8 comes from? The new code in CGAL likes >= 3.3.5 for one reason, but it is quite possible we wanted >= 3.5.8 for other reasons.

Reply:
Eigen 3.5.8 doen't even exist. I do not remember where this magic number came from...
Documentation modified on 6f2ec5ee

Reply:
Eigen version 3.5.8 doesn't even exist. I don't know where I took this number from :thinking: 
Modified with 3.3.5 on 6f2ec5e